src/dredd.js

```diff
@@ -104,6 +105,16 @@ https://dredd.readthedocs.io/en/latest/how-it-works/#using-https-proxy
     // Remove duplicate paths
     this.configuration.options.path = removeDuplicates(this.configuration.options.path);
 
+    if (this.configuration.options.require) {
+      let mod = this.configuration.options.require;
+      const abs = fs.existsSync(mod) || fs.existsSync(`${mod}.js`);
+      if (abs) {
+        mod = path.resolve(mod);
+      }
+      // eslint-disable-next-line global-require, import/no-dynamic-require
+      require(mod);
+    }
```

Comment:
Can we test and ensure whether Dredd is able to gracefully handle errors here?

- Dredd aborting with a sane message in case the path is wrong
- Dredd aborting with a sane massage in case the required module blows up (at least try/catch synchronously or something)